analyzer/src/funTest/assets/projects/synthetic/godep-expected-output.yml

```diff
@@ -0,0 +1,141 @@
+---
```

Comment:
commit-msg: consider "duration of the test" -> "duration of the test run"

Reply:
If have replace "duration" with "runtime" instead.